.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+    ignore = W503,E122,E203
+    max-line-length = 99
+    per-file-ignores = __init__.py:F401
```

Comment:
Add a new line to the end of this file.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline